### PR TITLE
BigQuery: clarify docs for sending data via streaming inserts.

### PR DESCRIPTION
**Pull request by shollyman**
The way the insertAll contract is specified don't allow us to control
behavior of how we serialize data to json sufficiently to ensure that
numeric values are properly encoded as a stringified representation.

Internal background: b/140265930

### CODE REVIEW

File: google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java [hunk 1]
```diff
@@ -50,9 +50,11 @@
 
   /**
    * A Google Big Query row to be inserted into a table. Each {@code RowToInsert} has an associated
-   * id used by BigQuery to detect duplicate insertion requests on a best-effort basis. Please
-   * notice that data for fields of type {@link LegacySQLTypeName#BYTES} must be provided as a
-   * base64 encoded string.
+   * id used by BigQuery to detect duplicate insertion requests on a best-effort basis.
+   *
+   * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```
Comment (elharo) [inline google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java]: per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"

File: google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java [hunk 2]
```diff
@@ -124,8 +127,11 @@ public boolean equals(Object obj) {
     }
 
     /**
-     * Creates a row to be inserted with associated id. Please notice that data for fields of type
-     * {@link LegacySQLTypeName#BYTES} must be provided as a base64 encoded string.
+     * Creates a row to be inserted with associated id.
+     *
+     * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```
Comment (elharo) [inline google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java]: per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"

File: google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java [hunk 3]
```diff
@@ -135,8 +141,11 @@ public static RowToInsert of(String id, Map<String, ?> content) {
     }
 
     /**
-     * Creates a row to be inserted without associated id. Please notice that data for fields of
-     * type {@link LegacySQLTypeName#BYTES} must be provided as a base64 encoded string.
+     * Creates a row to be inserted without associated id.
+     *
+     * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```
Comment (elharo) [inline google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java]: per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"

File: google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java [hunk 4]
```diff
@@ -178,8 +187,11 @@ public Builder addRow(RowToInsert rowToInsert) {
     }
 
     /**
-     * Adds a row to be inserted with associated id. Please notice that data for fields of type
-     * {@link LegacySQLTypeName#BYTES} must be provided as a base64 encoded string.
+     * Adds a row to be inserted with associated id.
+     *
+     * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```
Comment (elharo) [inline google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java]: per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"

File: google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java [hunk 5]
```diff
@@ -202,8 +215,11 @@ public Builder addRow(String id, Map<String, ?> content) {
     }
 
     /**
-     * Adds a row to be inserted without an associated id. Please notice that data for fields of
-     * type {@link LegacySQLTypeName#BYTES} must be provided as a base64 encoded string.
+     * Adds a row to be inserted without an associated id.
+     *
+     * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```
Comment (elharo) [inline google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java]: per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"

**Comment by meredithslota** [comment]
@shollyman What's left to do here?

**Comment by chingor13** [comment]
We actually need to move (recreate this PR) in https://github.com/googleapis/java-bigquery. We moved the bigquery issues to the new split repo, but are unable to move the open PRs.

**Comment by meredithslota** [comment]
@stephaniewang526 can likely help resolve this when she returns.

**Comment by stephaniewang526** [comment]
Will migrate this PR to java-bigquery

**Comment by stephaniewang526** [comment]
migrated PR to https://github.com/googleapis/java-bigquery/pull/190. Closing this PR.

**Pull request closed without merging**